### PR TITLE
Trust & Security introduction

### PR DESCRIPTION
**Pull request by jburger**
# About

This is a post about our intention to be transparent with improvement initiatives around our security posture, a little bit of a call to action at the end to elicit some opinions to help shape things too.

# Pre-requisites

- [x] The draft is complete and this post is ready to be reviewed.
- [x] I'm familiar with the [writing for Octopus TL;DR](https://style.octopus.com/writing-at-octopus-tldr).
- [x] The PR build passes validation, and if it doesn't, I've checked the [common validations errors](https://style.octopus.com/writing-at-octopus-tldr#common-validation-errors) and none of those apply.
- [x] The images I've included follow the [rules for working with images](https://style.octopus.com/images#rules-for-screenshots).

## How to review this PR
- Please review this for quality, tone & clarity.
- Please check the title, description and overall messaging
- If there is a more suitable image I'd love suggestions

### CODE REVIEW

**Comment by robpearson** [comment]
Nice post @jburger!

**Comment by jburger** [comment]
Thankyou both - have a great weekend :tada:

File: blog/2020-08/how-secure-is-octopus-deploy-anyway/index.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
+---
+title: How secure is Octopus Deploy, anyway?
+description: Octopus Deploy is maturing as a business, and that means our security posture is maturing too.
+author: jim.burger@octopus.com
+visibility: public
+published: 2999-08-10
```
Comment (wordlee) [inline blog/2020-08/how-secure-is-octopus-deploy-anyway/index.md:6]:
```suggestion
published: 2020-08-31
```

Comment (wordlee) [reply blog/2020-08/how-secure-is-octopus-deploy-anyway/index.md:6]: Scheduling the post

File: blog/2020-08/how-secure-is-octopus-deploy-anyway/index.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
+---
+title: How secure is Octopus Deploy, anyway?
+description: Octopus Deploy is maturing as a business, and that means our security posture is maturing too.
+author: jim.burger@octopus.com
+visibility: public
```
Comment (wordlee) [inline blog/2020-08/how-secure-is-octopus-deploy-anyway/index.md:5]:
```suggestion
visibility: private
```

File: blog/2020-08/how-secure-is-octopus-deploy-anyway/index.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+published: 2999-08-10
+metaImage: blogimage-security.png
+bannerImage: blogimage-security.png
+tags:
+ - Security
+---
+
+![A stylized Octopus & Shield icon](blogimage-security.png)
+
+It doesn’t take much research to conclude that most IT departments out there really care about the security of the tools they use. Using SaaS products is usually an exercise in trust, and Octopus Cloud is no different. You trust us to keep your data safe, and we trust you to use our platform [fairly](https://octopus.com/legal/acceptable-usage). So it’s a fair question, just how secure are we?
```
[the remark this answers is not in the document]
Comment (wordlee) [reply blog/2020-08/how-secure-is-octopus-deploy-anyway/index.md:15]: +1 let's change `it's a fair question` to `it's a reasonable question`.

File: blog/2020-08/how-secure-is-octopus-deploy-anyway/index.md [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+published: 2020-08-31
+metaImage: blogimage-security.png
+bannerImage: blogimage-security.png
+tags:
+ - Security
+---
+
+![A stylized Octopus & Shield icon](blogimage-security.png)
+
+It doesn’t take much research to conclude that most IT departments out there really care about the security of the tools they use. Using SaaS products is usually an exercise in trust, and Octopus Cloud is no different. You trust us to keep your data safe, and we trust you to use our platform [fairly](https://octopus.com/legal/acceptable-usage). So it’s a fair question, just how secure are we?
```
Comment (wordlee) [inline blog/2020-08/how-secure-is-octopus-deploy-anyway/index.md:15]:
```suggestion
It doesn’t take much research to conclude that most IT departments out there really care about the security of the tools they use. Using SaaS products is usually an exercise in trust, and Octopus Cloud is no different. You trust us to keep your data safe, and we trust you to use our platform [fairly](https://octopus.com/legal/acceptable-usage). So it’s a reasonable question, just how secure are we?
```

**Review by wordlee — APPROVED** [review]

**Pull request merged**